### DISCUSSION TITLE
Display link

### DISCUSSION THREAD

**Discussion by Zambieslar** [hyprwm/Hyprland#4273] [Q&A] [answered]
I know that this question has been asked a lot, but is there going to be a future release of Hyprland that contains the display link patch for wlroots?

**Comment by vaxerski** [comment]
dont think we can do that as we dont patch wlr.

**Comment by Zambieslar** [comment]
Ah, I thought it was already patched.... my bad, I must've misread the
gitlab page.

**Reply by vaxerski** [reply, marked as answer]
if it's a wlroots commit that was recently pushed, then yes, whenever I update the dep you will get it.
